aws_jupyter_proxy/awsproxy.py

```diff
@@ -150,7 +150,11 @@ def __init__(
             self.upstream_auth_info.service_name,
             self.upstream_auth_info.region,
         )
-        self.whitelisted_services = os.getenv("AWS_JUPYTER_PROXY_WHITELISTED_SERVICES")
+        self.whitelisted_services = (
+            os.getenv("AWS_JUPYTER_PROXY_WHITELISTED_SERVICES").strip(",").split(",")
+            if os.getenv("AWS_JUPYTER_PROXY_WHITELISTED_SERVICES") is not None
+            else None
```

Comment:
If make this `else []` then the check below doesn't have to do the `is not None` check since it will always be an array

Reply:
None is different from [] here. This whitelist is “opt-in” for now - If the env variable is not specified then there is no whitelist (None). If the env variable is specified then the whitelist takes effect ([] => nothing is whitelisted). os.getenv returns None if there is no environment variable. If its specified but empty, it returns “"

As discussed offline, adding a comment.